pkg/remote/crd_upgrade_test.go

```diff
@@ -0,0 +1,100 @@
+package remote_test
+
+import (
+	"testing"
+
+	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
+	"github.com/kyma-project/lifecycle-manager/pkg/remote"
+	"github.com/stretchr/testify/assert"
+	v1extensions "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+//nolint:funlen
+func TestShouldPatchRemoteCRD(t *testing.T) {
+	t.Parallel()
+	type args struct {
+		runtimeCrdGeneration      int64
+		kcpCrdGeneration          int64
+		kymaKcpCrdAnnotationValue string
+		kymaSkrCrdAnnotationValue string
+		err                       error
+	}
+	tests := []struct {
+		name string
+		args args
+		want bool
+	}{
+		{
+			name: "Different Skr Generation",
+			args: args{
+				err:                       nil,
```

Comment:
If you follow the order of each field in your args struct definition, you don't need to redeclare each fields name here.